apdx-validation.tex

```diff
@@ -1076,3 +1076,26 @@ \subsubsection*{Rules for the \class{Attachment} class}
 
 \printValid{The \sbol{hash} property of an \sbol{Attachment} is OPTIONAL and MAY contain a \sbol{String}.\\ Reference: \sec{sec:Attachment}}
 }
+
+\twothreezero{
+\subsubsection*{Rules for the \class{ExperimentalData} class} 
+\setcounter{sbolCtr}{13301}
+
+\printValid{An \sbol{ExperimentalData} MUST NOT have properties other than the following: \sbol{identity}, \\
+\sbol{persistentIdentity}, \sbol{displayId}, \sbol{version}, \sbol{wasDerivedFroms}, \sbol{wasGeneratedBys}, \sbol{name},\\
+\sbol{description}, and \sbol{annotations}.\\ Reference: \sec{sec:ExperimentalData}}
+}
+
+\twothreezero{
+\subsubsection*{Rules for the \class{Experiment} class} 
+\setcounter{sbolCtr}{13401}
+
+\printValid{An \sbol{Experiment} MUST NOT have properties other than the following: \sbol{identity}, \\
+\sbol{persistentIdentity}, \sbol{displayId}, \sbol{version}, \sbol{wasDerivedFroms}, \sbol{wasGeneratedBys}, \sbol{name},\\
+\sbol{description}, \sbol{annotations}, and \sbol{experimentalData}.\\ Reference: \sec{sec:Attachment}}
+
+\printValid{The \sbol{experimentalData} property of an \sbol{Experiment} is REQUIRED and MUST contain one or more \sbol{URI}s.\\ Reference: \sec{sec:Experiment}}
```

Comment:
I notice this is different than what is proposed in the SEP. The SEP says this field is optional. Is this an issue?